package.json

```diff
@@ -66,7 +66,7 @@
     "kbn:watch": "node scripts/kibana --dev --logging.json=false",
     "build:types": "tsc --p tsconfig.types.json",
     "docs:acceptApiChanges": "node --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
-    "kbn:bootstrap": "node scripts/register_git_hook",
+    "kbn:bootstrap": "yarn build:types && node scripts/register_git_hook",
```

Comment:
`Because dependent projects make use of .d.ts files that are built from their dependencies, you’ll either have to check in certain build outputs or build a project after cloning it before you can navigate the project in an editor without seeing spurious errors. `https://www.typescriptlang.org/docs/handbook/project-references.html#caveats-for-project-references

We can create a dedicated config for this though